vars/runBenchmarks.groovy

```diff
@@ -14,7 +14,7 @@ def call(String artifacts = null) {
             }
 
             stage('Run Benchmarks') {
-                List<String> mvnOptions = ['test', '-Dbenchmark']
+                List<String> mvnOptions = ['test', '-P jmh-benchmark']
```

Comment:
will it work with a space?

Reply:
yes.

Reply:
I would expect it is all going down via bash so likely yes - but I have fixed it as it should be a separate arg :)